<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRunModel</class>
 <widget class="QDialog" name="QtRunModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日镜的运动模式</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>140</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>俯仰电机运转方式</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="RB_Y_UP">
       <property name="text">
        <string>俯仰向上</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Y_DOWN">
       <property name="text">
        <string>俯仰向下</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_Y_STOP">
       <property name="text">
        <string>俯仰停止</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>130</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>水平电机运转方式</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="RB_X_SHUN">
       <property name="text">
        <string>顺时针</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_X_NIE">
       <property name="text">
        <string>逆时针</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RB_X_STOP">
       <property name="text">
        <string>俯仰停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="PB_Set_Save">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Set_Save_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtRunModel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_Set_Save_2</sender>
   <signal>clicked()</signal>
   <receiver>QtRunModel</receiver>
   <slot>saveCfg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveCfg()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
